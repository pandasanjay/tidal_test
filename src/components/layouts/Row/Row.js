import React from "react"

const Row = (props) => {
    return <>{props.children}</>
}
export default Row