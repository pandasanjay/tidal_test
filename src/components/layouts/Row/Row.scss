.grid-row {
    background: #000;
}