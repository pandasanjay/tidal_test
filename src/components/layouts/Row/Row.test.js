import React from "react";
import { render } from "@testing-library/react";
import Row from "./Row";
describe("Row Layout component ", () => {
  it("should render and check the snapshot", () => {
    const { asFragment } = render(<Row />);
    expect(asFragment()).toMatchSnapshot();
  });
});
