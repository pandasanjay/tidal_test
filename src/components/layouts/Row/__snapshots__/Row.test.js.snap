// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row Layout component  should render and check the snapshot 1`] = `<DocumentFragment />`;
